libs/styles/typography.css

```diff
@@ -129,6 +132,27 @@
   text-transform: uppercase;
 }
 
+.code-l {
+  font-size: var(--type-code-l-size);
+  line-height: var(--type-code-l-lh);
+}
+
+.code-m {
+  font-size: var(--type-code-m-size);
+  line-height: var(--type-code-m-lh);
+}
+
+.code-s {
+  font-size: var(--type-code-s-size);
+  line-height: var(--type-code-s-lh);
+}
+
+.code-l,
+.code-m,
+.code-s {
+  font-weight: var(--type-code-all-weight);
+}
+
```

Comment:
Would it be better to put these in `code.css`? In typography even though minimal, still adds bytes to pages.

Reply:
Maybe? Consonant has typography for code which we have not implemented, so I thought it best to add it the same way the other typography is defined. See surrounding code and https://consonant.adobe.com/1975e5ba1/p/216c7e-typography/t/244b88

It can be argued that any `<pre><code>` on page should have typography styling, including the font, but that's potentially a high impact CSS change and loading another font all the time would definitely be slower.

Reply:
🤷  Either way for me, it is 33 (including vars below) lines of CSS, so not a lot. If it follows consonant and this is the best spot for it, why not.

Reply:
I made https://github.com/adobecom/milo/pull/1187 to address this separately.